modules/db/src/main/java/org/mule/module/db/internal/domain/connection/OracleDbConnection.java

```diff
@@ -99,32 +104,33 @@ public Struct createStruct(String typeName, Object[] attributes) throws SQLExcep
         return super.createStruct(typeName, attributes);
     }
 
-    @Override
-    protected void resolveLobs(String typeName, Object[] attributes) throws SQLException
+    protected void resolveLobs(String typeName, Object[] attributes, TypeResolver typeResolver) throws SQLException
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add the "@Override" annotation above this method signature ([squid:S1161](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1161))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7681&issues=AWjzWMEwotIXnmsUaGus&open=AWjzWMEwotIXnmsUaGus)

[ISSUE_KEY]: AWjzWMEwotIXnmsUaGus